.gitignore

```diff
@@ -1,2 +1,4 @@
 .import/
-build/*
\ No newline at end of file
+build/*
+burrito-fg/target/*
+burrito-fg/Cargo.lock
```

Comment:
We should be including the Cargo.lock file in git. It contains helpful information like the package checksum and exact package version.